EncodeQueryParams.pas

```diff
@@ -0,0 +1,29 @@
+unit EncodeQueryParams;
+
+interface
+
+type
+  TEncodeQueryParams = class
```

Comment:
Indy has a 'TIdURI` class which have a `EncodeParams` method or something like it, that already does this.